instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -183,11 +189,16 @@ def process_request(self, request):
             carrier_getter = wsgi_getter
             collect_request_attributes = wsgi_collect_request_attributes
 
-        token = attach(extract(carrier, getter=carrier_getter))
-
+        token = context = None
+        span_kind = SpanKind.INTERNAL
+        if get_current_span() is INVALID_SPAN:
+            context = extract(request_meta, getter=wsgi_getter)
```

Comment:
earlier we were using `carrier_getter` which could be either a wsgi or asgi getter. Now we are always using `wsgi_getter`. is this intentional?

Reply:
Oh my bad. corrected to `carrier_getter`.